### ISSUE TITLE
How to make a test that uses other modules?

### ISSUE THREAD

**Issue by danfuzz** [flow-typed/flow-typed#613]
I'm trying to add a definition for the module `express-ws`. Unsurprisingly, many of the types used by this module are defined by the module `express`. Unfortunately, if I write this in my test file:

```javascript
import { $Application, $Request } from "express";
```

then I get the following complaint when I try to run the test:

```
ERROR: express-ws_v2.x.x/flow_v0.37.x-
 * express-ws_v2.x.x/flow_v0.37.x- (flow-v0.37.1): Unexpected Flow errors(2):
   0-test_express-ws.js:3
     3: import { $Application, $Request } from "express";
                                               ^^^^^^^^^ express. Required module not found
```

It's unclear what I should do to convince the test harness that there's an `express` module to import.

**Comment by danfuzz** [comment]
In case you're curious, the error I elided in the report above is a different unexpected import failure, which prompted me to file <https://github.com/facebook/flow/issues/3180>.

**Comment by Brianzchen** [comment]
Would be solved by #1857

**Comment by GAntoine** [comment]
Closing as duplicate.
